client/my-sites/plans/plans.jsx

```diff
@@ -0,0 +1,28 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import config from 'config';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import getIntervalTypeForTerm from 'state/plans/get-interval-type-for-term';
+import getCurrentPlanTerm from 'state/selectors/get-current-plan-term';
```

Comment:
~~You can `import { getCurrentPlanTerm } from 'state/selectors';` instead.`~~
It feels a little odd to import a non-selector from the global selectors directory.